src/app/Library/CrudPanel/Traits/Relationships.php

```diff
@@ -117,6 +117,33 @@ public function parseRelationFieldNamesFromHtml($fields)
         return $fields;
     }
 
+    /**
+     * Gets the relation fields that DON'T contain the provided relations.
+     *
+     * @param  string|array  $relations  - the relations to exclude
+     * @param  bool  $include_nested  - if the nested relations of the same relations should be excluded too.
+     */
+    private function getRelationFieldsWithoutRelationType($relations, $include_nested = false)
```

Comment:
This, for example, is a GREAT name. I didn't even read the code, because the name and parameters explain PERFECTLY what it does.